tests/models/data/horovod/train_default_model.py

```diff
@@ -0,0 +1,36 @@
+import argparse
```

Comment:
is this meant to be a (unit)test because by the name it won't be found
why there is `data/horovod/` would it rather be `tests/models/script_train_horovod.py `

Reply:
Added a docstring at the top for clarity.  This script is meant to be executed from `test_horovod.py`.  Reason for this is to test driving the training via `horovodrun` using multiple parallel worker processes.